sparkle/src/components/CardButton.tsx

```diff
@@ -69,15 +66,21 @@ export function CardButton({
   rel = "",
   replace,
   shallow,
+  ...props
 }: CardButtonProps) {
   const { components } = React.useContext(SparkleContext);
   const Link: SparkleContextLinkType = href ? components.link : noHrefLink;
 
+  const cardButtonClassNames = cn(
+    cardButtonVariants({ variant, size }),
+    className
+  );
+
   if (href) {
```

Comment:
Hot take: can we remove this feature of optionally passing a href to the CardButton and limit its scope to a simple card that does smth when hovered and another thing when clicked? This component lowkey feels like two very different components at once.

Reply:
AFAICT the version of the CardButton where we pass a href is not used in front.

Reply:
Well not sure to follow why does it feel like two components. One could want to open a link when the card is linked right ? AFAICT it basically behaves as a regular button

Reply:
just pointing out that the subsets of props that are used between the two behaviors seem a little disjoint, if I give an `onMouseEnter` and an `href` the former will be ignored. In order to share the style maybe having two components, one with a link and another that is clickable, would be more readable